### PR TITLE
Adding the DiElectron Trigger SF (HLT_Ele17_Ele12_CaloIdL_TrackIdL_IsoVL_)

### PR DESCRIPTION
**Pull request by sdevissc**
The Trigger SF are obtained from, https://twiki.cern.ch/twiki/bin/viewauth/CMS/EgHLTScaleFactorMeasurements, more precisely from https://indico.cern.ch/event/492286/contribution/2/attachments/1226439/1795635/TSG_10022016.pdf , DoubleEG Trigger SF are given. The numbers are inserted in a json file to be read like the DoubleMuon Trigger SF json.

### CODE REVIEW

File: data/ScaleFactors/El17El12_SF.json [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+          "bin": [
+            1.2, 
+            2.4
+          ], 
+          "error_low": 0.023, 
+          "value": 0.989, 
+          "error_high": 0.023
+        } 
+      ]
+    },
```
Comment (blinkseb) [inline data/ScaleFactors/El17El12_SF.json]: Spurious comma here, please try the file before doing the PR

**Comment by blinkseb** [comment]
There's one spurious comma, but otherwise the file is good.

Thanks!

**Comment by swertz** [comment]
Just one question: where did you find the numbers? I don't see them on the page...

Also, are the SFs in https://github.com/cp3-llbb/Framework/blob/CMSSW_7_6_3p/data/ScaleFactors/Mu17Mu8_SF.json from the same (TT) analysis?

(Asking this because adding MuE/EMu would be interesting for us)

**Comment by sdevissc** [comment]
The numbers can be found in the pdf which I linked in the first comment, there are on slide 13 (the pdf has been shown in TSG, and is linked from the twiki mentioned above, sorry for the confusion).
Btw I just did a test and the json file seems ok now.

**Comment by blinkseb** [comment]
Thanks. Build failure is unrelated to this PR and will need to be fixed with another PR.

**Comment by swertz** [comment]
Sorry to get back to this @sdevissc, but in the PDF the numbers given have only 2 bins in |eta| while your jsons have 4... I'd like to add the MuE paths, could you please tell me how you got the 4-bin ones?

**Pull request merged**